tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -58,6 +60,8 @@ export class CardFobControllerComponent {
   @Input() highestStep!: number;
   @Input() lowestStep!: number;
   @Input() showExtendedLine?: Boolean = false;
+  @Input() isProspectiveFobFeatureEnabled: Boolean = false;
+  @Input() prospectiveStep: number | null = null;
```

Comment:
nitpick: Why make this null rather than `undefined`?
```suggestion
  @Input() prospectiveStep?: number;
```

Reply:
prospectiveStep will be set to null on mouseleave event. While undefined means the variable is not defined, assigning it to null makes more sense to me.
Is this reasonable to you? Or any other strong reason to use undefined?

I noticed this two inputs should be optional. Adding question marks.